conda/models/index_record.py

```diff
@@ -81,30 +56,100 @@ class Link(DictSafeMixin, Entity):
 EMPTY_LINK = Link(source='')
 
 
-class FeaturesField(ListField):
+def push_individual_feature(result_map, val):
+    k, v = translate_feature_str(val)
+    result_map[k] = v
+
+
+def _make_provides_features(track_features, instance):
```

Comment:
I actually added the python and numpy features when we were thinking about using features for build groups.  I don't think they're as important now.  I'll back that out.  Except... I do want to figure something out for python at some point, so that we can have `python=pypy2.7` for example.